### PR TITLE
 argpartition #16292 frontends_Numpy_Sorting PR

### PR DESCRIPTION
**Pull request by MelatKebedeAbraham**

### CODE REVIEW

**Comment by Sam-Armstrong** [comment]
Close https://github.com/unifyai/ivy/issues/16292

File: ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py [hunk 1]
```diff
@@ -33,3 +33,7 @@ def sort_complex(a):
 @to_ivy_arrays_and_back
 def lexsort(keys, /, *, axis=-1):
     return ivy.lexsort(keys, axis=axis)
+
+@to_ivy_arrays_and_back
+def argpartition(a, kth, axis=-1, kind='introselect', order=None):
+    return ivy.argpartition(a, kth, axis=axis, kind=kind)
```
Comment (Sam-Armstrong) [inline ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py:39]: ivy has no 'argpartition' function. Please find a different function, or combination of functions, which achieve the same result as numpy.argpartition

**Review by Sam-Armstrong — CHANGES_REQUESTED** [review]
@MelatKebedeAbraham thanks for the PR! 😊

Please can you make changes to address my comment below? Thanks!

File: ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py (hunk 1 above)
Comment (MelatKebedeAbraham) [reply ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py:39]: Hi, @Sam-Armstrong Thank you for your message. I have two questions regarding the argpartition function. Firstly, I would like to know whether I should use the numpy methods or ivy methods when combining different functions to implement the argpartition method. Secondly, I would appreciate it if you could provide me with information on where to find the implemented methods of ivy.

Comment (Sam-Armstrong) [reply ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py:39]: @MelatKebedeAbraham yes, all the functions should be ivy functions as this is part of the frontend api. I'd recommend reading through these two pages in the docs for more info about this 😀:

https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html
https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html

To your second question, all the functions in the ivy functional api are contained within the files in this directory and its sub-directories: https://github.com/unifyai/ivy/tree/master/ivy/functional/ivy

Feel free to let me know if you have more questions!

Comment (MelatKebedeAbraham) [reply ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py:39]: @Sam-Armstrong Thank you for your guidance and support. I appreciate your help and will definitely reach out if I have any further questions.

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by Sam-Armstrong** [comment]
@MelatKebedeAbraham please could you open a different PR if you're changing the function you're implementing? Thanks 🙂

**Comment by MelatKebedeAbraham** [comment]
@Sam-Armstrong am currently trying to implement the experiment argpartition in order to call it here.

**Pull request closed without merging**